cloud-sql-proxy/cloud-sql-proxy.sh

```diff
@@ -313,68 +332,21 @@ function run_validation() {
   fi
 }
 
-function configure_hive_warehouse_dir() {
-  # Wait for master 0 to create the metastore db if necessary.
-  run_with_retries run_validation
```

Comment:
It used to be that masters 1 and 2 in HA clusters would call configure_hive_warehouse_dir, which would call run_validation, which would call systemctl restart hive-metastore. Now the metastore will never be restarted on masters 1 and 2. We should pull restarting the metastore out of the run_validation function (since it didn't make sense there in the first place) and move it down into main, making sure we do it for all 3 masters

Reply:
Added call for `run_with_retries run_validation` explicitly for 1 and 2 masters.